### ISSUE TITLE
FP: oelint.vars.pkgspecific.FILES for bbappends

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#216]
`openssl_%.bbappend` with
```
FILES_libcrypto += "${libdir}/libcrypto.so.1"
```
gives a false positive `oelint.vars.pkgspecific.FILES:Variable FILES is package-specific and therefore it should be FILES_${PN}`.

But only if the bbappend is passed without a matching bb file
